### PR TITLE
Prepare 0.6.1 release

### PR DESCRIPTION
**Pull request by johannes-b**

### CODE REVIEW

File: content/docs/0.6.0/installation/setup-keptn/index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Select one of the following options:
   - [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 
 2. Create AKS cluster
-  - Master version >= `1.12.x` (tested version: `1.12.8`)
+  - Master version >= `1.13.x` (tested version: `1.14.8`)
```
Comment (christian-kreuzberger-dtx) [inline content/docs/0.6.0/installation/setup-keptn/index.md]: Did it work with 1.14.8? I thought we need at least 1.15.5?

File: content/docs/quickstart/setup_platform/setup_aks/index.md [hunk 2]
```diff
@@ -9,5 +9,5 @@ keywords: setup
   - [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 
 ## 2. Create AKS cluster
-  - Master version >= `1.12.x` (tested version: `1.12.8`)
+  - Master version >= `1.13.x` (tested version: `1.14.8`)
```
Comment (christian-kreuzberger-dtx) [inline content/docs/quickstart/setup_platform/setup_aks/index.md]: Same question as above for AKS

File: content/docs/0.6.0/installation/setup-keptn/index.md (hunk 1 above)
Comment (johannes-b) [reply content/docs/0.6.0/installation/setup-keptn/index.md]: So far it is working.

Comment (johannes-b) [reply content/docs/0.6.0/installation/setup-keptn/index.md]: Today I'm having trouble with 1.14.8. I will now test 1.15.5.

**Pull request merged**